src/com/clover/remote/client/transport/websocket/WebSocketCloudCloverTransport.ts

```diff
@@ -105,6 +105,8 @@ export class WebSocketCloudCloverTransport extends WebSocketCloverTransport {
             function(data) { this.deviceNotificationSent(data);}.bind(this),
             function(error) {
                 this.connectionError(this.webSocket, "Error sending alert to device." + error);
+                // This may end a reconnect attempt
```

Comment:
maybe add a comment at the top of the initialize() method discussing the setting of reconnect to false in an error condition.

Reply:
Yep, did so